pkg/domain/infra/abi/containers.go

```diff
@@ -484,13 +484,14 @@ func (ic *ContainerEngine) ContainerCheckpoint(ctx context.Context, namesOrIds [
 		err  error
 		cons []*libpod.Container
 	)
-	checkOpts := libpod.ContainerCheckpointOptions{
-		Keep:           options.Keep,
-		TCPEstablished: options.TCPEstablished,
-		TargetFile:     options.Export,
-		IgnoreRootfs:   options.IgnoreRootFS,
-		KeepRunning:    options.LeaveRunning,
-	}
+	checkOpts := libpod.ContainerCheckpointOptions{}
```

Comment:
Why did you change the structure?

Reply:
Because there are lots of different between checkpoint and restore. With more Features or changes, the code would be not clear. So I want to distinguish between them. If it has other reason to keep one struct, or just not necessary for the pr, I can change it back.